website/docs/tutorials/07-testing/03-fuzzing/strategies.md

```diff
@@ -28,8 +28,10 @@ func test_integers{syscall_ptr : felt*, range_check_ptr}(a : felt, b : felt) {
 ```
 
 :::note
-By default, Protostar applies the [`strategy.felts()`](#strategyfelts) strategy to all felt
-parameters.
+In the past, Protostar used to apply the [`strategy.felts()`](#strategyfelts) strategy to all felt
```

Comment:
I think we shouldn't mention the past in the docs. We should just describe it as it is. It makes it easier to read.

Reply:
[Address direct comments](https://github.com/software-mansion/protostar/pull/995/commits/d53af654b1789f34245a4e363740e3035ff1cfb5)